SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -322,6 +324,9 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/**
 	  * This will enable warning messages for ActorSpawning that could be legitimate but is likely to be an error.
 	  */
-	UPROPERTY(Config)
+	UPROPERTY(EditAnywhere, Config, Category="Logging")
```

Comment:
driveby

Reply:
This is just used for allowing specific "destroy on bad creation" logic that MW were relying upon. Was there a reason for making this more public?

Reply:
The primary change I wanted to make here was to put it in a Category, and the EditAnywhere was just for consistency with the rest of the file.  Happy to remove either/both of these changes if you'd prefer.

Reply:
changes to this line reverted

Reply:
Thanks!